<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="3" total="6" passed="2">
  <reporter-output>
    <line>
      <![CDATA[No priority5<br>]]>
    </line>
    <line>
      <![CDATA[This is nopriority3<br>]]>
    </line>
    <line>
      <![CDATA[This is priority4<br>]]>
    </line>
    <line>
      <![CDATA[Retrying2<br>]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="106" started-at="2022-11-10T22:54:01 EST" finished-at="2022-11-10T22:54:01 EST">
    <groups>
      <group name="nopriority">
        <method signature="HomeTest.homeQuote3()[pri:3, instance:HomeTest.HomeTest@150c158]" name="homeQuote3" class="HomeTest.HomeTest"/>
      </group> <!-- nopriority -->
      <group name="home">
        <method signature="HomeTest.homeQuote4()[pri:0, instance:HomeTest.HomeTest@150c158]" name="homeQuote4" class="HomeTest.HomeTest"/>
        <method signature="HomeTest.homeQuote5()[pri:0, instance:HomeTest.HomeTest@150c158]" name="homeQuote5" class="HomeTest.HomeTest"/>
        <method signature="HomeTest.homeQuote3()[pri:3, instance:HomeTest.HomeTest@150c158]" name="homeQuote3" class="HomeTest.HomeTest"/>
      </group> <!-- home -->
    </groups>
    <test name="Test" duration-ms="106" started-at="2022-11-10T22:54:01 EST" finished-at="2022-11-10T22:54:01 EST">
      <class name="HomeTest.HomeTest">
        <test-method status="PASS" signature="homeQuote5()[pri:0, instance:HomeTest.HomeTest@150c158]" name="homeQuote5" duration-ms="34" depends-on-groups="homeQuote" started-at="2022-11-10T22:54:01 EST" finished-at="2022-11-10T22:54:01 EST">
          <reporter-output>
            <line>
              <![CDATA[No priority5<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- homeQuote5 -->
        <test-method status="PASS" signature="homeQuote3()[pri:3, instance:HomeTest.HomeTest@150c158]" name="homeQuote3" duration-ms="4" started-at="2022-11-10T22:54:01 EST" finished-at="2022-11-10T22:54:01 EST">
          <reporter-output>
            <line>
              <![CDATA[This is nopriority3<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- homeQuote3 -->
        <test-method status="FAIL" signature="homeQuote4()[pri:0, instance:HomeTest.HomeTest@150c158]" name="homeQuote4" duration-ms="4" depends-on-groups="home" started-at="2022-11-10T22:54:01 EST" finished-at="2022-11-10T22:54:01 EST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at HomeTest.HomeTest.homeQuote4(HomeTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[This is priority4<br>]]>
            </line>
            <line>
              <![CDATA[Retrying2<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- homeQuote4 -->
      </class> <!-- HomeTest.HomeTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
